test/sciml_style.jl

```diff
@@ -293,23 +313,18 @@
                    3 => 4)
     """
 
-    formatted_str1 = raw"""
-    Dict{Int, Int}(# Comment
-        1 => 2,
-        3 => 4)
-    """
-
-    formatted_str2 = raw"""
+    # Now aligns to opening paren
```

Comment:
Why? The point of `variable_call_indent` is that it does not align with opening parentheses.

Reply:
Is it supposed to be this?

```
    formatted_str = raw"""
    Dict{Int, Int}(# Comment
       1 => 2,
       3 => 4)
    """
```

Reply:
Yes. Without a comment, YAS style indenting only allows this:
```julia
    formatted_str = raw"""
    Dict{Int, Int}(1 => 2,
                   3 => 4)
    """
```
With `variable_call_indent`, **additionally** this is allowed:
```julia
    formatted_str = raw"""
    Dict{Int, Int}(
       1 => 2,
       3 => 4)
    """
```
Adding a comment at the end of a line should not change the indentation of the following lines.